installers/win/editpath/EditPath.iss

```diff
@@ -0,0 +1,165 @@
+; Copyright (C) 2021-2023 by Bill Stewart (bstewart at iname.com)

```

Comment:
do we need to worry about this?

Reply:
No.  The EditPath project is LGPL licensed.  That particular file is just an example of how to use EditPath in a setup script.  Not needed at all by Medley, so I will delete it.

But we should probably add an open-source software attribution "page" to my medley scripts and to online.  I will put that on my list of things to do.